static/app/views/performance/traces/content.tsx

```diff
@@ -155,85 +168,126 @@ function TraceRow({trace}: {trace: TraceResult<Field>}) {
+            {t('Span Duration')}
+          </StyledPanelHeader>
+          <StyledPanelHeader align="right" lightText>
+            {t('Issues')}
+          </StyledPanelHeader>
+
+          {spans.map(span => (
+            <SpanRow key={span.id} span={span} trace={trace} />
+          ))}
+        </SpanPanelContent>
+      </StyledPanel>
+    </SpanTablePanelItem>
+  );
+}
+
+function SpanRow({span, trace}: {span: SpanResult<Field>; trace: TraceResult<Field>}) {
+  const theme = useTheme();
   return (
     <Fragment>
-      <StyledPanelItem align="right">
+      <StyledSpanPanelItem align="right">
         <SpanIdRenderer
           projectSlug={span.project}
           transactionId={span['transaction.id']}
           spanId={span.id}
-          trace={trace}
+          traceId={trace.trace}
           timestamp={span.timestamp}
         />
-      </StyledPanelItem>
-      <StyledPanelItem align="left">
+      </StyledSpanPanelItem>
+      <StyledSpanPanelItem align="left">
         <Description>
           <ProjectRenderer projectSlug={span.project} hideName />
           <strong>{span['span.op']}</strong>
           <em>{'\u2014'}</em>
           {span['span.description']}
         </Description>
-      </StyledPanelItem>
-      <StyledPanelItem align="right">
-        <EmptyValueContainer>{'\u2014'}</EmptyValueContainer>
-      </StyledPanelItem>
-      <StyledPanelItem align="right">
+      </StyledSpanPanelItem>
+      <StyledSpanPanelItem align="right">
+        <TraceBreakdownContainer>
+          <SpanBreakdownSliceRenderer
+            sliceName={span.project}
+            sliceStart={Math.ceil(span['precise.start_ts'] * 1000)}
+            sliceEnd={Math.floor(span['precise.finish_ts'] * 1000)}
+            trace={trace}
+            theme={theme}
+          />
+        </TraceBreakdownContainer>
+      </StyledSpanPanelItem>
+      <StyledSpanPanelItem align="right">
         <PerformanceDuration milliseconds={span['span.duration']} abbreviation />
-      </StyledPanelItem>
-      <StyledPanelItem align="right">
+      </StyledSpanPanelItem>
+      <StyledSpanPanelItem align="right">
         <EmptyValueContainer>{'\u2014'}</EmptyValueContainer>
-      </StyledPanelItem>
+      </StyledSpanPanelItem>
     </Fragment>
   );
 }
 
 type SpanResult<F extends string> = Record<F, any>;
 
-interface TraceResult<F extends string> {
+export interface TraceResult<F extends string> {
+  breakdowns: TraceBreakdownResult[];
   duration: number;
+  end: number;
   name: string | null;
   numSpans: number;
   spans: SpanResult<F>[];
+  start: number;
   trace: string;
 }
 
+interface TraceBreakdownBase {
+  end: number;
+  start: number;
+}
+
+type TraceBreakdownProject = TraceBreakdownBase & {
+  kind: 'project';
+  project: string;
+};
+
+type TraceBreakdownMissing = TraceBreakdownBase & {
+  kind: 'missing';
+  project: null;
+};
+
+export type TraceBreakdownResult = TraceBreakdownProject | TraceBreakdownMissing;
```

Comment:
There's also the `unknown` type that can be at the tail of a breakdown in the case that we didn't load enough transactions to compute the full breakdown.

Reply:
Ah true, I haven't seen one yet, I'll add it in another PR when we find a trace w/ one